app/server/src/main/scala/org/bitcoins/server/WalletRoutes.scala

```diff
@@ -226,10 +200,8 @@ case class WalletRoutes(wallet: AnyDLCHDWalletApi, node: Node)(implicit
           reject(ValidationRejection("failure", Some(exception)))
         case Success(BroadcastDLCFundingTx(eventId)) =>
           complete {
-            wallet.getDLCFundingTx(eventId).flatMap { tx =>
-              node.broadcastTransaction(tx).map { _ =>
-                Server.httpSuccess(tx.txIdBE.hex)
-              }
+            wallet.broadcastDLCFundingTx(eventId).map { tx =>
```

Comment:
just to make sure I have this straight, the actual node broadcast is now handled by the wallet?

Reply:
the wallet has a function that just calls `nodeApi.broadcastTransaction`